### ISSUE TITLE
Circles' top/bottom anchors are incorrect

### ISSUE THREAD

**Issue by matthew-e-brown** [johannes-wolf/cetz#252]
`top` and `bottom` are swapped.

MWE:

```typst
#import "@preview/cetz:0.1.2"

#set page(width: auto, height: auto, margin: 1cm)

#cetz.canvas(length: 0.5cm, {
    import cetz.draw: *

    circle((0, 0), name: "bob", radius: 5, fill: black, stroke: none)

    let anchors = (
        "top",      "top-left",     "top-right",
        "bottom",   "bottom-left",  "bottom-right",
        "center",   "left",         "right",
    )

    for pos in anchors {
        content(
            "bob." + pos,
            padding: 0.35,
            frame: "rect",
            fill: rgb(255, 255, 255, 100%, stroke: 1pt + black),
            text(size: 0.75em, pos)
        )
    }
})
```

![From 12 o'clock, clockwise: 'bottom', 'top-right', 'right', 'bottom-right', 'top', 'bottom-left', 'left', 'top-left', and back to 'bottom'.](https://github.com/johannes-wolf/cetz/assets/43387454/ba6cb8ee-f049-4d09-a84d-d0a28c37ec85)

Looks to be from here, top and bottom should have `+/-` `ry` swapped:

https://github.com/johannes-wolf/cetz/blob/282e400b4e0ff583f2a3cabbffa507b94b6f53db/src/draw.typ#L552-L569

**Comment by johannes-wolf** [comment]
Fun thing is, there are even tests checking this but I merged reference images with the anchors swapped :facepalm:

**Comment by johannes-wolf** [comment]
This got fixed in 0.2.0. Anchors changed their names to compass coordinates.
